tests/workflow_test.go

```diff
@@ -152,6 +152,68 @@ func (s *integrationSuite) TestStartWorkflowExecution_TerminateIfRunning() {
 	s.Equal(enumspb.WORKFLOW_EXECUTION_STATUS_RUNNING, descResp.WorkflowExecutionInfo.Status)
 }
 
+func (s *integrationSuite) TestStartWorkflowExecutionWithDelay() {
+	id := "integration-start-workflow-with-delay-test"
+	wt := "integration-start-workflow-with-delay-test-type"
+	tl := "integration-start-workflow-with-delay-test-taskqueue"
+	identity := "worker1"
+
+	startDelay := 5 * time.Second
```

Comment:
I don't know, 5 sec seems a bit long. 3 maybe?
Up to you, I don't have strong opinion.

Reply:
Can we test signal with start as well?